pymc3/tests/helpers.py

```diff
@@ -77,3 +80,13 @@ def match_value(self, k, dv, v):
         else:
             result = dv.find(v) >= 0
         return result
+
+
+def select_by_precision(float64, float32):
+    """Helper function to choose reasonable decimal cutoffs for different floatX modes."""
+    decimal = float64 if theano.config.floatX == "float64" else float32
+    return decimal
+
+
+def fxarray(x):
```

Comment:
How about on the next iteration of fixes?  I say this because this whole procedure is going to take several iterations of fixes anyway.